.run/Connector Consumer Corp.run.xml

```diff
@@ -1,12 +1,13 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="Connector Consumer Corp" type="Application" factoryName="Application">
-    <option name="ALTERNATIVE_JRE_PATH" value="temurin-22" />
+    <option name="ALTERNATIVE_JRE_PATH" value="temurin-21" />
```

Comment:
why not to set it to 23? LTS is not a requirement for a project that meant to stay up-to-date